src/index.js

```diff
@@ -18,7 +18,7 @@ var autorun = require('./autorun');
 var server = require('./server');
 var socketHandlers = require('./socket_handlers');
 var ffi = require('./ffiimp');
-//var rc = require('./rc');
+var rc = require('./rc');
```

Comment:
Requires an package.json update to 0.0.8.

Reply:
Also, can we use the optional require here in case the package doesn't install nice?